src/Answer.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class Answer extends Component {
+  
+  render() {
+    let {title, score} = {...this.props.data};
+
+    return (
+      
+      <label > 
+        <input type='checkbox' 
```

Comment:
We likely only want users to be able to select a single answer per question. Let's change this from a checkbox to a radio element. Radio elements can also take a `name` attribute that will limit our selection to a single answer. See example [here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_radio).

The name should be unique per question. i.e. all of the answer elements in question one might have a name attribute equal to answer-group-1.